src/Symfony/Bundle/FrameworkBundle/Resources/config/console.php

```diff
@@ -206,6 +207,13 @@
             ])
             ->tag('console.command')
 
+        ->set('console.command.messenger_count', CountMessagesCommand::class)
```

Comment:
Note you should probably remove this definition from the `FrameworkBundleExtension` if the class does not exists, [since Fwb 6.2+ can be installed with Messenger <6.2 where the command would not exist](https://github.com/symfony/symfony/runs/6724535303?check_suite_focus=true#step:8:2246)

Reply:
I've fixed this.

Reply:
@ogizanagi , I don't know if it's what you meant but I added a condition on https://github.com/symfony/symfony/pull/46571/files#diff-5e6c5fa1a5d0246cea4c25f663a78b0a76654ea006db0b2c4f462220437880bcR340

Reply:
> I'm not sure why I must do it as my commit adds it

because you add it in the 6.2 version of FrameworkBundle and Messenger.
But you could have Fwb 6.2 installed without Messenger 6.1. Then the command class does not exist.

You should rather remove the definition from the FrameworkBundleExtension class, though.

Reply:
@ktherage : I've pushed to your branch the changes regarding this :)

Reply:
> because you add it in the 6.2 version of FrameworkBundle and Messenger.
> But you could have Fwb 6.2 installed with Messenger 6.1. Then the command class does not exist.

Thanks for the explanation, it's clearer I didn't mind it when opening the PR.

> @ktherage : I've pushed to your branch the changes regarding this :)

Thanks for making it in the right way. Mine was kind a clumsily done as I had no idea what I was doing here :sweat_smile:.